add-ons/management-infra/management-infra.addon

```diff
@@ -0,0 +1,34 @@
+# Name: management-infra
+# Description: Creates management objects for ManageIQ, Cloudforms, etc
+# Required-Vars: 
+
+echo Creating management-infra project
+oc new-project management-infra --description="Management Infrastructure" --as=system:admin
+
+echo Creating Service Accounts: management-admin, inspector-admin
+oc create sa management-admin -n management-infra --as=system:admin
+oc create sa inspector-admin -n management-infra --as=system:admin
+
+echo Creating Cluster Roles: management-infra-admin, hawkular-metrics-admin
+# From 3.7 on, it can be done with oc create clusterrole. see https://bugzilla.redhat.com/show_bug.cgi?id=1476166
```

Comment:
Why is this comment relevant? Is this backwards compatible when 3.7.0 is used or is a new command a necessity to make this work on later versions?

It is better to add this to an improved addon when 3.7.0 is released and force the needed OpenShift version or record this as an issue: WDYT?

Reply:
Yes, it will be backward compatible when 3.7 is used. However, with 3.7, it will be straightforward and will not need a local addon file.

Reply:
It is needed anyway for this addon.

Reply:
leave the comment, but consider creating an issue to track this possible future update to the addon installation. WDYT?

Reply:
Seems pretty reasonable.